be/src/util/system_metrics.cpp

```diff
@@ -325,6 +357,9 @@ void SystemMetrics::_update_memory_metrics() {
     }
 #endif
 
+    _update_datacache_mem_tracker();
+    _update_pagecache_mem_tracker();
```

Comment:
update memtracker only when collecting metrics?

Reply:
Yes, every 15 seconds, the daemon thread will trigger `system_mem_metrics` hook to collect metrics.